WooCommerce/Resources/en.lproj/Localizable.strings

```diff
@@ -134,15 +139,9 @@
 /* Order refunded shipping label title. The string variable shows the shipping label service name (e.g. USPS). */
 "%@ label refund requested" = "%@ label refund requested";
 
-/* Review title. Reads as {Review author} left a review on {Product}. */
-"%@ left a review on %@" = "%1$@ left a review on %2$@";
```

Comment:
Comes from the unit tests, so shouldn't be sent for translation. 🔥 

https://github.com/woocommerce/woocommerce-ios/blob/0073cca68e5a6bc351b51227780d1ce415ae8bac/WooCommerce/WooCommerceTests/Reviews/ReviewViewModelTests.swift#L76-L80

Reply:
You can exclude the unit tests when scanning the code… but tbh, ideally developers should not use `NSLocalizedString` in Unit Tests at all anyway.

1. `NSLocalizedString` should only be used for strings you want to end up in the `.strings` file. That's literally the purpose of this macro. To be detected by `genstrings`.
    - The macro just ends up calling `bundle.localizedString(forKey:,value:)` in the end, except that using `NSLocalizedString` allows you to be detected by `genstrings` and provide a comment (which is not used at all by the `localizedString(…)` method and the API and does not even end up in your source file, that comment param is not used by the implementation of `NSLocalizedString` and its only purpose is for when `genstrings` parses the source code with its internal regex to extract it…)
    - So really, devs should only use `NSLocalizedString` for `genstrings` to pick it up, otherwise they should use `Bundle.localizedString(…)` directly. That's actually what, for example, `AppLocalizedString` ends up calling as well.
2. In the context of unit tests, the translation for copy is likely not going to be translated anyway — especially if this is a Unit Test, the unit test bundle won't have the `*.lproj/Localizable.strings` available to it anyway.

So, while you can still update the tooling's setup to exclude `WooCommerceTests/`, to me the root of the issue here is that the tests are using `NSLocalizedString` in the first place where they should not.

Reply:
Indeed.

Some of those [were _intentionally_ localized](https://github.com/woocommerce/woocommerce-ios/commit/bb18b4357f3e90e4a8fdb65f476bd2871b05e62b). 

It's useful to run UI and snapshot tests in different locales, but I can't think of a reason why we'd want to run the _unit_ tests on different locales. Something worth enquiring about...